package.json

```diff
@@ -105,6 +113,7 @@
   },
   "eslintIgnore": [
     "modernizr-custom.js",
+    "language-file-to-csv.js",
```

Comment:
Why ignore this?

Reply:
Since adding that package-lock I keep getting [this error](https://github.com/shift72/core-template/runs/5730066747?check_suite_focus=true). Any ideas? I tried for a long time but no success adding the prettier ignore option.

Reply:
I've just pushed a fix into develop so you can merge that in and see if it it's okay... strange that running the linter locally doesn't pick that up but running in the Github Action does.  Also strange that we haven't noticed it before!

Reply:
Oh sweet. Nice one. Merged develop into this branch and reverted ignore

Reply:
Do you need to change this:
`"js:lint": "prettier '{scripts,site/static/js}/**/!(modernizr-custom).js' --write && eslint ./site/static/js/*.js ./scripts --fix"`

to this:
`"js:lint": "prettier '{scripts,site/static/js}/**/!(modernizr-custom).*js' --write && eslint ./site/static/js/*.js ./scripts --fix"`

so the linter will pick up that mjs file

Reply:
Ah yep that was a good idea. Cheers added.